src/bundlers/node/genezioRuntimeHandlerGenerator.ts

```diff
@@ -0,0 +1,243 @@
+/* eslint-disable no-useless-escape */
+
+export const genezioRuntimeHandlerGenerator = (className: string): string => `
+/** This is an auto generated code. This code should not be modified since the file can be overwritten
+ *  if new genezio commands are executed.
+ */
+
+import {  ${className.replace(/["]/g, "")} as genezioClass } from "./module.mjs";
```

Comment:
why is this necessary?

Reply:
I copied this from the lambda generator. @radu1122 @costinsin any input?